test/Interop/Cxx/members/emit-function-from-static-var-init.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop -Xfrontend -validate-tbd-against-ir=none)
+//
+// REQUIRES: executable_test
+
+// TODO: See why -validate-tbd-against-ir=none is needed here
```

Comment:
Are you planning to fix this before the patch lands? If not can you make sure to create an SR and reference it here?

Reply:
Done. I created an SR for it.